COCOAPODS.md

```diff
@@ -187,15 +228,365 @@ import cocoapods.subspec_dependency.*
+    * `packageName` – to specify the package name. If you specify it, you can import the library using the package name: `import cocoapods.<packageName>`
+
+2. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+   
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("new_pod") {
+               extraOpts = listOf("-compiler-option")
+               packageName = "package"
+           }
+
+       }
+   }
+   ```
+
+   </div>
+
+3. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+If you use the `packageName` parameter, you can import the library using the package name: `import cocoapods.<packageName>`:
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.packageName
+```
+
+</div>
+
+### Add a dependency on a static Pod library
+
+You can add dependencies on a static Pod library with `pod()` and `useLibraries()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of the library using the `pod()` function.
+
+2. Call the `useLibraries()` function: it enables a special flag for static libraries. 
+
+   > You can add a dependency on a static library from a repository or stored locally.
+   {:.note}
+
+3. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("YandexMapKit") {
+               version = "~> 3.2"
+           }
+           useLibraries()
+
+       }
+   }
+   ```
+
+   </div>
+
+
+4. Re-import the project.
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+### Update Podfile for Xcode
+
+Kotlin requires Podfile changes to work correctly with Xcode:
```

Comment:
This step requires only if user has his own Xcode project. If Xcode project exists and `podfile` is changed to actual path, absence of this information will fail `podInstall` task with diagnostic message about possible lost paths in`Podfile`

Reply:
Updated the intro and add an additional sentence on error messages.